File: stress/t/int_sumsq.f
      PROGRAM INTSUMSQ
      INTEGER*8 S,I,N,IALT
      S=0
      I=1
      N=1000
   10 IF (I.GT.N) GOTO 20
      S=S+I*I
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'sumsq=',S
      IALT=0
      I=1
   30 IF (I.GT.N) GOTO 40
      IALT=IALT+(-1_8)*I
      S=S-I
      I=I+1
      GOTO 30
   40 CONTINUE
      WRITE(*,'(A,I0)') 'alt=',IALT
      END

File: stress/t/int_sumsq.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t s=0,i=1,n=1000,alt=0;
  while(i<=n){ s=s+i*i; i=i+1; }
  printf("sumsq="); printf("%ld",(long)s); printf("\n");
  alt=0; i=1;
  while(i<=n){ alt=alt+(-1)*i; s=s-i; i=i+1; }
  printf("alt="); printf("%ld",(long)alt); printf("\n");
  return 0;
}
